test/testinput/bufr_ncep_adpupa_nc002001.yaml

```diff
@@ -0,0 +1,148 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# 
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+observations:
+  - obs space:
+      name: bufr
+
+#     obsdatain: "./testinput/gdas.t12z.adpupa_thinned.tm00.bufr_d"
+      obsdatain: "NC002001"
+
+      exports:
+        group_by_variable: pressure
+        variables:
+          # MetaData
+          timestamp:
+            datetime:
+              year: "NC002001/YEAR"
+              month: "NC002001/MNTH"
+              day: "NC002001/DAYS"
+              hour: "NC002001/HOUR"
+              minute: "*/MINU"
```

Comment:
Does MINU need NC002001?

Reply:
NO. The NC002001 subset has no info about MINU variable.